tools/roslaunch/src/roslaunch/config.py

```diff
@@ -407,14 +407,16 @@ def _select_machine(self, node):
             # assign to local machine
             return self.machines['']            
 
-def load_config_default(roslaunch_files, port, roslaunch_strs=None, loader=None, verbose=False, assign_machines=True):
+def load_config_default(roslaunch_files, port, roslaunch_args=None, roslaunch_strs=None, loader=None, verbose=False, assign_machines=True):
```

Comment:
Please add new arguments at the end of the signature since existing code might call keyword arguments as positional which would break.

Same below.